packages/labs/task/src/task.ts

```diff
@@ -10,8 +10,7 @@ export interface TaskFunctionOptions {
   signal?: AbortSignal;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type TaskFunction<D extends ReadonlyArray<unknown>, R = any> = (
+export type TaskFunction<D extends ReadonlyArray<unknown>, R = unknown> = (
```

Comment:
I tacked this on to this PR instead of making a 7-char PR. It's technically breaking, but I don't see why we should default to `any` in the cases TS can't infer the task type. `unknown` is much safer.